setup.py

```diff
@@ -34,13 +34,25 @@
 
 with open(".dep-versions") as f:
     lines = f.readlines()
-    jax_version = [line[4:].strip() for line in lines if "jax=" in line][0]
-    pl_str = "pennylane="
-    pl_str_length = len(pl_str)
-    pl_version = [line[pl_str_length:].strip() for line in lines if pl_str in line][0]
+    jax_version = next(line[4:].strip() for line in lines if "jax=" in line)
+    pl_version = next((line[10:].strip() for line in lines if "pennylane=" in line), None)
+    lq_version = next((line[10:].strip() for line in lines if "lightning=" in line), None)
+
+pl_min_release = 0.35
+lq_min_release = pl_min_release
+
+if pl_version is not None:
+    pennylane_dep = f"pennylane @ git+https://github.com/pennylaneai/pennylane@{pl_version}"
+else:
+    pennylane_dep = f"pennylane>={pl_min_release}"
+if lq_version is not None:
+    lightning_dep = f"pennylane-lightning=={lq_version}"  # use TestPyPI wheels to avoid rebuild
```

Comment:
How are you specifying `TestPyPI` here?

Reply:
It is done in the Makefile unfortunately, no way to do it in setup.py

But the version specifier is only available on TestPyPI.

Reply:
Awesome! Thanks!